src/util-print.c

```diff
@@ -227,6 +227,7 @@ void PrintStringsToBuffer(uint8_t *dst_buf, uint32_t *dst_buf_offset_ptr, uint32
                         src_buf[ch] == '\n' ||
                         src_buf[ch] == '\r') ? (uint8_t)src_buf[ch] : '.');
     }
+    dst_buf[dst_buf_size - 1] = 0;
```

Comment:
Do you have a bt of a ASAN error for this? Would like to see if this issue might have a deeper cause.

(thought I already posted a comment like this, but can't find it)